### ISSUE TITLE
In-config visibility oddities on Nested branch

### ISSUE THREAD

**Issue by asaka-wa** [WeakAuras/WeakAuras2#3566]
### Description

Selecting an Aura/group in config used to toggle off the visibility of all other Auras not specifically toggled to be visible. That no longer happens and Auras stay visible throughout. 

### WeakAuras Version

Nested branch 

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [ ] Yes
- [X] No

### Lua Error

n/a

### Reproduction Steps

1. Reload
2. open config
3. select an Aura
4. See all others staying visible
5. use master button to toggle all off
6. select an Aura
7. select another Aura
8. see both selected and previously selected Auras are still visible

### Last Good Version

pre-nested I think

**Comment by asaka-wa** [comment]
Hmm, visibility doesn't look quite right yet. 

1. Open config and all currently loaded Auras are shown - fine
2. Select an Aura and all Auras hide except for the selected Aura - great!
3. Select another Aura... and now both Auras are still visible - bad

Further to that, the eye icons aren't lit for Auras that remain shown in this way. And clicking the "master" eye icon on Loaded, to show all, and clicking again to hide all, doesn't hide the ones that have remained shown in this way.

**Comment by InfusOnWoW** [comment]
I can't reproduce that behaviour. If I select a different aura it gets hidden unless the eye is fully open.

**Comment by asaka-wa** [comment]
Ah, specifically this happens with groups. 

I made a couple of Auras outside of any groups to show the behaviour I see and it behaved as you intend. 

Then I clicked around my other Auras (all in groups) and see the behaviour I described above. 

I also notice that selecting a group. Then selecting a regular Aura. If I then click the same regular Aura again to deselect it, then REselect it again, everything else hides. 

https://user-images.githubusercontent.com/54901700/168589885-91d2bd94-f0f1-47d4-97a6-6eb1aba64641.mp4
